zebra-chain/src/orchard/tests/vectors/group_hash.rs

```diff
@@ -6,7 +6,7 @@ pub struct TestVector {
     pub(crate) point: [u8; 32],
 }
 
-// From https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/orchard_group_hash.py
+// From <https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/orchard_group_hash.py>
```

Comment:
Optional: not needed.
```suggestion
// From https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/orchard_group_hash.py
```

Reply:
I fixed this one in [36fc845](https://github.com/ZcashFoundation/zebra/pull/4494/commits/36fc8453d4239291deb6e3f692a0d6b9253f17e8).